src/commands/guard/index.ts

```
@@ -39,27 +40,71 @@ export const executeInteraction = async (
+        await editCampChannelExecute(interaction);
+        break;
+      case roleStatsCommand.name:
+        await roleStatsExecute(interaction);
+        break;
+      default:
+        console.error(`Command ${commandName} not found`);
+        break;
+    }
+  } catch (error) {
+    // Handle Discord API errors gracefully
+    const discordError = error as any; // Type assertion for the error
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)